### PR TITLE
Lighter hover colour for sidebar for #297

### PR DESCRIPTION
**Pull request by mic-smith**

### CODE REVIEW

File: src/sidebars/favourites/favourites.less [hunk 1]
```diff
@@ -26,6 +26,10 @@
       background-color: @highlight-colour;
     }
   }
+
+  .darkens:hover {
+    background: #1F262E;
```
Comment (bjedrzejewski) [inline src/sidebars/favourites/favourites.less]: Can we use a variable here? If one does not exist for this color, let's create one.

Comment (owennw) [inline src/sidebars/favourites/favourites.less]: Interestingly the other change uses a `fade`, but we don't want one here?

Comment (mic-smith) [inline src/sidebars/favourites/favourites.less]: Sure. Will pull out a variable and I think there's at least one other colour in that file that's not using one too. Will fix them up too.

The favourites needed an override for the style as the fade version with 0.5 alpha meant the green of the drop-area style showed through for the hover style.

Comment (owennw) [inline src/sidebars/favourites/favourites.less]: That makes sense.

**Comment by mic-smith** [comment]
Closing as target branch needs changing

**Pull request closed without merging**